src/Extensions/ExceptionReporter.php

```diff
@@ -174,22 +192,44 @@ protected function createStackFrame(array $entry): StackFrame
 
         $frame = new StackFrame($filename, $function, $lineno);
 
-        if ($filename !== "{no file}") {
-            $this->loadContext($frame, $filename, $lineno, 5);
+        if ($this->root_path && strpos($filename, $this->root_path) !== -1) {
+            $frame->abs_path = $filename;
+            $frame->filename = substr($filename, strlen($this->root_path));
+        }
 
-            if ($this->root_path && strpos($filename, $this->root_path) !== -1) {
-                $frame->abs_path = $filename;
-                $frame->filename = substr($filename, strlen($this->root_path));
+        if ($this->isBlacklisted($filename)) {
+            $frame->context_line = "### BLACKLISTED ###";
```

Comment:
I kind of want to call it something else. Maybe `### FILTERED FILE ###` instead. Because if you just see "BLACKLISTED" in your log, you might not understand what it means. You just can't see the stacktrace, and only see that message.

Reply:
Good point, I will update the terminology :-)